Packages/com.unity.inputsystem/InputSystem/Editor/Settings/InputSettingsProvider.cs

```diff
@@ -126,11 +126,20 @@ public override void OnGUI(string searchContext)
 
                 m_SupportedDevices.DoLayoutList();
 
+                EditorGUILayout.LabelField("iOS", EditorStyles.boldLabel);
```

Comment:
A bit dunno tbh, is it worth putting platform specific detail into input system settings panel versus placing it into player settings where we already have established platform specific settings pattern? If I would be a user I would expect something changing the plist to be in the player settings, not input system.

Not to stop the PR, just want clarify what UX we expect here.

Reply:
We don't put package settings in Player Settings, as far as I am aware.